File: logfeeder.py
#!/usr/bin/env python
#
# logfeeder.py
#
# reads a specified file and outputs it in delayed chunks
#

import sys
import random
import time

if sys.argv < 1: 
  print 'Usage logfeeder.py <input>'

lines = file(sys.argv[1]).readlines()
for line in lines:
  print line.strip()
  sys.stdout.flush()
  time.sleep(random.random() * 5)

File: logfeeder.rb
#!/usr/bin/env ruby
#
# logfeeder.rb
#
# reads a specified file and outputs it in delayed chunks
#

if ARGV.length < 1
  puts "Usage: logfeeder.rb <input>"
  exit 1
end

File.open(ARGV[0]) do |file|
    lines = file.readlines
    lines.each do |line|
      $stdout.write line
      $stdout.flush
      sleep rand # * 2 
    end
end

